js/interface.js

```diff
@@ -2257,6 +2262,17 @@ $('#fl-store-2fa-select, #fl-ent-2fa-select').on('change', function (e) {
   socket.to(socketClientId).emit('aab.apple.login.2fa.device', e.target.value);
 });
 
+// When app is loaded we check if user previosly saved 'demo user' but not 'demo user password'.
+Fliplet().then(function () {
+  checkDemoUser();
```

Comment:
I believe `checkDemoUser()` should run right after the form is populated. There should be a place in the code where data is retrieved via the APIs and then the form is populated with previously entered values.

Reply:
But it wouldn't work if a user has auto-fill from google like in my case on the screenshot. Therefore if we check only after getting data from API, then it may lead to an error because the demo user will be filled, but not from the API.

Reply:
@YaroslavOvdii can we disable the auto-fill for this? I don't think we need it as it can lead to more problems in the process.

What I am suggesting:
1. disable browser autofill
2. after loading the data via the APIs and fields are set to their values, run the `checkDemoUser()` function to update the required status
3. on email input blur, `checkDemoUser()` (you already have this, no changes required)

cc @tonytlwu

Reply:
@squallstar 
I was able to trick google auto-fill but not totally disabled it.
I tried deferent ways with the autocomplete attribute but it didn't work.
Even try to give google fake input for auto-fill it won't buy it. 
So I did it not so great in my opinion.
The idea was to take data from API and check if the demo user is empty or not. 
If it's empty we clearing the input field.

Reply:
We might need to consider the presence of password managers as well. Some of them might autofill.

Therefore we should run checks at the following points:

1. `Fliplet().then()` to check its original state
1. `.on('input change')` to check when the data has changed
    - Probably don't need to chec `.on('blur')` if we do this
    - This should also check for when data is loaded from other APIs

Reply:
@tonytlwu 
.on('input change') to check when the data has changed
This should apply for the demo user field only or for all inputs with type password and email?
Because of Google and other password managers targeting these types of fields in DOM.

Reply:
@YaroslavOvdii Just for the demo user field please.

Reply:
@tonytlwu 
Made changes as was requested.